### ISSUE TITLE
[det_mv3_db] Very slow evaluation speed when there are hundreds of bboxes/polygons in prediction

### ISSUE THREAD

**Issue by pharrellyhy** [PaddlePaddle/PaddleOCR#6292]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：`Ubuntu 18.04`
- 版本号/Version：Paddle：`paddlepaddle-gpu==2.2.2.post111`  PaddleOCR：`paddleocr==2.5.0.3` 
- 问题相关组件/Related components：
- 运行指令/Command Code：`python tools/train.py -c configs/det/det_mv3_db_custom.yml -o Global.pretrained_model=./pretrain_models/MobileNetV3_large_x0_5_pretrained.pdparams`
- 完整报错/Complete Error Message：**Very slow evaluation speed when there are hundreds of bboxes/polygons in image.**

Basically I'm using the same config file as `det_mv3_db.yaml`, only `img_size`, `batch_size` and `num_workers` are changed but I don't think it's related to my issue. All train and val images are of size (1920, 2560, 3) and there are hundreds of annotations (word/sentence) for each of them.

The issue I have is that when the model outputs hundreds of bboxes/polygons, it takes very long time (~10s) to do the evaluation of a single image. Take below screenshot as a reference.

<img width="483" alt="Screen Shot 2022-05-14 at 9 55 13 PM" src="https://user-images.githubusercontent.com/16664753/168430488-ac57da5d-30be-445f-bf3b-73053105d779.png">

I use below code ([https://github.com/PaddlePaddle/PaddleOCR/blob/7db347787eb858115a39a1423ea9d209b32e261b/tools/program.py#L466](url)) to get the evaluation time.
```
start = time.time()
post_result = post_process_class(preds, batch_numpy[1])
print(f"** post result shape: {post_result[0]['points'].shape}")
eval_class(post_result, batch_numpy)
print(f'** eval class time: {time.time() - start}')
````

I don't know how to improve it at this moment, so any help is appreciated. Thanks!

**Comment by WenmuZhou** [comment]
Large image size will bring slow model forward process, too much text will reduce the speed of calculating metrics

**Comment by WenmuZhou** [comment]
when testing，you can resize long side of image to 1280

**Comment by pharrellyhy** [comment]
Seems this is the expected bahavior when doing metric calculation. Closing now...
